deepchecks/vision/suites/default_suites.py

```diff
@@ -293,8 +293,23 @@ def data_integrity(image_properties: List[Dict[str, Any]] = None,
     )
 
 
-def full_suite(**kwargs) -> Suite:
-    """Create a suite that includes many of the implemented checks, for a quick overview of your model and data."""
+def full_suite(n_samples: Optional[int] = 5000, **kwargs) -> Suite:
+    """Create a suite that includes many of the implemented checks, for a quick overview of your model and data.
+
+    Parameters
+    ----------
+    n_samples : Optional[int] , default : 5000
+        Number of samples to use for the checks in the suite. If None, all samples will be used.
+
+    Returns
+    -------
+    Suite
+        A suite that includes integrity checks.
+    """
+    args = locals()
+    args.pop('kwargs')
+    non_none_args = {k: v for k, v in args.items() if v is not None}
```

Comment:
What items can be here except n_samples? Why not write it explicitly?

Reply:
Youre right. Added the rest of them